Formula/ignition-sensors4.rb

```diff
@@ -1,9 +1,9 @@
 class IgnitionSensors4 < Formula
   desc "Sensors library for robotics applications"
   homepage "https://bitbucket.org/ignitionrobotics/ign-sensors"
-  url "https://bitbucket.org/ignitionrobotics/ign-sensors/get/e77b3a1e2abcf4b4ac1fad2158cf89c59792e6ec.tar.gz"
-  version "3.999.999~0~20200214~e77b3a1"
-  sha256 "5761a7b63acbcda868b042814a1e91cc9e0962a429f40e77c2bff918cd3e8bb3"
+  url "https://bitbucket.org/ignitionrobotics/ign-sensors/get/84f8018bc3185ada6f4e85a8d0728f2cdd3b3699.tar.gz"
+  version "3.999.999~0~20200214~84f8018"
```

Comment:
I recommend updating the date portion of the version as well to ensure it is increasing compared to the previous version

Reply:
oops updated. a8e206b1377200cb116a01b3c61a19b692f5d7a0